src/main/java/net/mcreator/workspace/elements/VariableElement.java

```diff
@@ -18,10 +18,16 @@
 
 package net.mcreator.workspace.elements;
 
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
 public class VariableElement implements IElement {
+	private static final Map<VariableElementType, String> variables = new HashMap<>();
 
 	private String name;
-	private VariableElementType type;
+	private String type;

```

Comment:
What happens with existing variables in workspace due to this change?

Reply:
Current variables are saved with a String, so I made this change to be compatible with old workspaces.

Reply:
But they are probably saved as uppercase, not lowercase string like now? Also please don't resolve yourself as I then don't notice your comments. Thanks!

Reply:
No, both are lower case strings.
Before/Now:
![image](https://user-images.githubusercontent.com/38427877/116956049-49364d00-ac62-11eb-9964-7a7fdbf4a67a.png)

This PR:
![image](https://user-images.githubusercontent.com/38427877/116956070-58b59600-ac62-11eb-986e-11bd97f7354d.png)

Reply:
Oh, you are right, my bad. Fine then ;)